### ISSUE TITLE
Error: incorrect header check

### ISSUE THREAD

**Issue by stdex** [avocode/avocode#598]
When I try to upload only one PSD.
Ubuntu 14.04 x64

It's looks like issue #592 

Also I can't reproduce it again.

-----
App v.2.5.0
Source UID: 48220
```
Error: incorrect header check
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by jankuca** [comment]
Hello, yes, it is the same issue. Thanks for reporting it from the beta.

Does reuploading the design work?

**Comment by stdex** [comment]
Loading was successful. The application does not was closed. It is only notification.

**Comment by jankuca** [comment]
OK, good to hear. The app tries re-downloading and re-unpacking files corrupted by this bug.

**Comment by rychlis** [comment]
As this is duplicate of #447, I am closing this issue. Thanks for the report

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
